cmd/alertmanager-authorization-server/Dockerfile

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/alpine:3.17
+FROM gcr.io/distroless/static-debian12
```

Comment:
Since this implies the `latest` tag, what would be the plan to avoid stale cache problem on the node? I saw they publish images with some hashes as tags.

Reply:
I had no plan. I saw that they are not tagging their images, so I assumed Google knows best and each of these images would be built exactly once and then never changed.

More realistically, we wipe our Docker mirror in the CI cluster every week, plus the script to build these images uses `docker build --pull --no-cache`, so I guess we're fine?

Reply:
While I dislike the "Google knows best" attitude, I don't have facts to prove otherwise. The size of unpacked image is 1.99MB so `docker build --pull --no-cache` sounds OK.

/lgtm